src/sql-parser/tests/testdata/show

```diff
@@ -37,154 +37,198 @@ SHOW ROLES
+SHOW SINKS FROM foo.bar
+=>
+ShowObjects(ShowObjectsStatement { object_type: Sink, from: Some(UnresolvedObjectName([Ident("foo"), Ident("bar")])), in_cluster: Some(Unresolved(Ident("baz"))), extended: false, full: false, materialized: false, filter: None })
+
+parse-statement
+SHOW SINKS IN CLUSTER baz
+----
+SHOW SINKS
+=>
+ShowObjects(ShowObjectsStatement { object_type: Sink, from: None, in_cluster: Some(Unresolved(Ident("baz"))), extended: false, full: false, materialized: false, filter: None })
 
 parse-statement
 SHOW TABLES LIKE '%foo%'
 ----
 SHOW TABLES LIKE '%foo%'
 =>
-ShowObjects(ShowObjectsStatement { object_type: Table, from: None, extended: false, full: false, materialized: false, filter: Some(Like("%foo%")) })
+ShowObjects(ShowObjectsStatement { object_type: Table, from: None, in_cluster: None, extended: false, full: false, materialized: false, filter: Some(Like("%foo%")) })
 
 parse-statement
 SHOW FULL VIEWS
 ----
 SHOW FULL VIEWS
 =>
-ShowObjects(ShowObjectsStatement { object_type: View, from: None, extended: false, full: true, materialized: false, filter: None })
+ShowObjects(ShowObjectsStatement { object_type: View, from: None, in_cluster: None, extended: false, full: true, materialized: false, filter: None })
 
 parse-statement
 SHOW SOURCES
 ----
 SHOW SOURCES
 =>
-ShowObjects(ShowObjectsStatement { object_type: Source, from: None, extended: false, full: false, materialized: false, filter: None })
+ShowObjects(ShowObjectsStatement { object_type: Source, from: None, in_cluster: None, extended: false, full: false, materialized: false, filter: None })
 
 parse-statement
 SHOW MATERIALIZED SOURCES FROM foo
 ----
 SHOW MATERIALIZED SOURCES FROM foo
 =>
-ShowObjects(ShowObjectsStatement { object_type: Source, from: Some(UnresolvedObjectName([Ident("foo")])), extended: false, full: false, materialized: true, filter: None })
+ShowObjects(ShowObjectsStatement { object_type: Source, from: Some(UnresolvedObjectName([Ident("foo")])), in_cluster: None, extended: false, full: false, materialized: true, filter: None })
 
 parse-statement
 SHOW MATERIALIZED VIEWS FROM foo LIKE '%foo%'
 ----
 SHOW MATERIALIZED VIEWS FROM foo LIKE '%foo%'
 =>
-ShowObjects(ShowObjectsStatement { object_type: View, from: Some(UnresolvedObjectName([Ident("foo")])), extended: false, full: false, materialized: true, filter: Some(Like("%foo%")) })
+ShowObjects(ShowObjectsStatement { object_type: View, from: Some(UnresolvedObjectName([Ident("foo")])), in_cluster: None, extended: false, full: false, materialized: true, filter: Some(Like("%foo%")) })
 
 parse-statement
 SHOW INDEXES FROM foo
 ----
 SHOW INDEXES FROM foo
 =>
+<<<<<<< HEAD
 ShowIndexes(ShowIndexesStatement { table_name: Name(UnresolvedObjectName([Ident("foo")])), extended: false, filter: None })
+=======
+ShowIndexes(ShowIndexesStatement { table_name: Some(UnresolvedObjectName([Ident("foo")])), in_cluster: None, extended: false, filter: None })
+>>>>>>> sql: support SHOW (INDEXES|SINKS)...IN CLUSTER
```

Comment:
I think you caught some merge conflicts here.

Reply:
Derp. Fixed